.github/workflows/auto-merge.yaml

```diff
@@ -1,16 +1,33 @@
 name: auto-merge
+concurrency:
+  cancel-in-progress: true
+  group: ${{github.workflow}}-${{github.event.pull_request.number}}
 on:
   pull_request:
-    types: [labeled, opened, reopened]
+    types: [labeled, opened, reopened, synchronize]
     branches:
       - 'master'
       - 'release-*'
+permissions:
+  contents: read
 jobs:
-  pr-mgmt:
-    uses: kumahq/.github/.github/workflows/wfc_pr_management.yml@main
-    with:
-      approve: true
-      merge: true
-      matchLabel: "ci/auto-merge"
-    secrets:
-      github-token: ${{ secrets.GITHUB_TOKEN }}
+  approve-and-auto-merge:
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'ci/auto-merge')
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Generate GitHub app token
+        id: github-app-token
+        uses: tibdex/github-app-token@3beb63f4bd073e61482598c45c71c1019b59b73a
+        with:
+          app_id: ${{ vars.KONG_MESH_APP_ID }}
```

Comment:
Should be `KONG_MESH_APP_ID` or just `APP_ID` and `APP_PRIVATE_KEY`?

Reply:
arg :( indeed I copy pasta